### ISSUE TITLE
DF6 - differentiation strategy

### ISSUE THREAD

**Issue by AbbyANoble** [StevenClontz/cal1-tbil-2021-checkit#3]
Would like if there were a few more situations to mix in. Currently it's the same three each time, so predictable patterns to learn. Also, there are no e^x or ln x included right now.

Using the pic below as an example, some other variants might be:

a) `trig (sqrt(poly))` to contrast with the current `sqrt(trig(poly))`; could also replace trig with natural log

b) `poly/(poly)^n` or replace one poly with an `e^x, e^f(x) or ln x`   (chain inside quotient, current is quotient inside power)

c) more product situations -- `(poly^n)*(trig),  (poly^n)*(e^f(x)), (poly^n)*(ln x)`

<img width="526" alt="Screen Shot 2021-09-14 at 10 09 08 PM" src="https://user-images.githubusercontent.com/85640990/133358791-019aede8-e314-4e11-be87-44c32ae420f1.png">

**Comment by StevenClontz** [comment]
I agree and have also opened this upstream (see link above). This is another contributed exercise so I'm not sure how easy it will be to update. I'll try posting the request in the Slack to see if someone else can/wants to take a stab at this.

**Comment by AbbyANoble** [comment]
I looked at the code, but I'm just not well-versed enough in sage to really fix it. We start that section next week, so I'll probably make a supplementary static set of practice problems for them with the hope of eventually using CheckIt for assessments.
